packages/foam-vscode/src/features/tags-tree-view/index.ts

```diff
@@ -83,15 +85,12 @@ export class TagsProvider implements vscode.TreeDataProvider<TagTreeItem> {
 type TagTreeItem = Tag | TagReference | TagSearch;
 
 export class Tag extends vscode.TreeItem {
-  constructor(
-    public readonly tag: string,
-    public readonly noteUris: vscode.Uri[]
-  ) {
+  constructor(public readonly tag: string, public readonly notes: Note[]) {
     super(tag, vscode.TreeItemCollapsibleState.Collapsed);
-    this.description = `${this.noteUris.length} reference${
-      this.noteUris.length !== 1 ? 's' : ''
+    this.description = `${this.notes.length} reference${
+      this.notes.length !== 1 ? 's' : ''
     }`;
-    this.tooltip = this.description;
+    this.tooltip = getContainsTooltip(notes);
```

Comment:
Ok, I thought of this crazy idea, I can't not share it :)

Here are a couple of considerations for tags, which might be interesting to think about.
When I look at a tag probably I want to know where it shows up in the target note, in what context.
What if we showed in the description the list of lines where the tag shows up in the file. e.g.

```
12: - I need to check whether this would work #follow-up
25: - Check-in with Max #follow-up
```

where `12` and `25` are the line numbers.


Might be out of scope for this PR, but thought it'd be good to share.

This wouldn't change anything in the way we deal with the preview for orphans.

Reply:
I like this idea, it could be very useful indeed!

I think it's outside the scope of the PR though, so I suggest we keep it as a future improvement.